crates/router/src/connector/netcetera.rs

```diff
@@ -313,6 +309,110 @@ impl
         types::authentication::AuthenticationResponseData,
     > for Netcetera
 {
+    fn get_headers(
+        &self,
+        req: &types::authentication::ConnectorAuthenticationRouterData,
+        connectors: &settings::Connectors,
+    ) -> CustomResult<Vec<(String, request::Maskable<String>)>, errors::ConnectorError> {
+        self.build_headers(req, connectors)
+    }
+
+    fn get_content_type(&self) -> &'static str {
+        self.common_get_content_type()
+    }
+
+    fn get_url(
+        &self,
+        req: &types::authentication::ConnectorAuthenticationRouterData,
+        connectors: &settings::Connectors,
+    ) -> CustomResult<String, errors::ConnectorError> {
+        let base_url = build_endpoint(self.base_url(connectors), &req.connector_meta_data)?;
```

Comment:
Validation is missing in `admin.rs` for the metadata struct which has the `merchant_endpoint_prefix`